src/pages/RegisterPage.js

```diff
@@ -24,3 +45,24 @@ export default () =>
       <Footer />
     </div>
   </div>);
+
+RegisterFormView.propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  i18n: PropTypes.func.isRequired,
+};
+
+export const RegisterForm = reduxForm({
+  form: 'RegisterForm', // a unique identifier for this form
+})(RegisterFormView);
+
+export const RegisterPageInstance = ({ i18n }) =>
+  <RegisterForm onSubmit={console.log} i18n={i18n} />;
+
+RegisterPageInstance.propTypes = {
+  i18n: PropTypes.func.isRequired,
+};
+
+export default connect(
+  state => ({ i18n: getTranslator(state) })
+)(RegisterPageInstance);
```

Comment:
You should remove `RegisterForm` and `RegisterPageInstance` and use `connectForm` directly.